src/routes/landing/components/FilterableLocationSelector.css

```diff
@@ -1,27 +1,10 @@
-@import url('../../../modules/common/constants/colors.css');
-
 :root {
   --hover-color: rgba(0, 0, 0, 0.1);
   --item-height: 30px;
   --list-margin: 30px;
```

Comment:
you need to move the --item-height definition to LocationSelector.css and you can omit --list-margin, I think.

Reply:
done